script/test

```diff
@@ -49,14 +49,17 @@ const resourcePath = CONFIG.repositoryRootPath
 let executablePath
 if (process.platform === 'darwin') {
   const executablePaths = glob.sync(path.join(CONFIG.buildOutputPath, '*.app'))
+  assert(executablePaths.length !== 0, `No atom build found. Please run "script/build" and try again.`)
   assert(executablePaths.length === 1, `More than one application to run tests against was found. ${executablePaths.join(',')}`)
   executablePath = path.join(executablePaths[0], 'Contents', 'MacOS', path.basename(executablePaths[0], '.app'))
 } else if (process.platform === 'linux') {
   const executablePaths = glob.sync(path.join(CONFIG.buildOutputPath, 'atom-*', 'atom'))
+  assert(executablePaths.length !== 0, `No atom build found. Please run "script/build" and try again.`)
   assert(executablePaths.length === 1, `More than one application to run tests against was found. ${executablePaths.join(',')}`)
   executablePath = executablePaths[0]
 } else if (process.platform === 'win32') {
   const executablePaths = glob.sync(path.join(CONFIG.buildOutputPath, '**', 'atom*.exe'))
+  assert(executablePaths.length !== 0, `No atom build found. Please run "script/build" and try again.`)
   assert(executablePaths.length === 1, `More than one application to run tests against was found. ${executablePaths.join(',')}`)
```

Comment:
I think we can refactor this to prevent copy-pasting the same thing three times.
```js
function assertexecutablePaths(executablePaths) {
	assert(executablePaths.length !== 0, `No atom build found. Please run "script/build" and try again.`)
  	assert(executablePaths.length === 1, `More than one application to run tests against was found. ${executablePaths.join(',')}`)
}
```